### ISSUE TITLE
Visual Builder : Transparent header background : Full width section : Full width module: Issue- visual builder tooltip is not clickable

### ISSUE THREAD

**Issue by rijendra** [elegantthemes/Divi-Beta#418]
### Problem:

The visual builder tooltip is not clickable of the full width header/slider module because of the z-index issue as the tooltip is below the transparent header so when mouse goes over the primary header to click the tooltip, it disappears, which makes its impossible to click on any buttons of tooltip.

### Steps To Reproduce:
1.  theme customizer > header & navigation > primary header > set the background transparent
2. New page  
3. Go to visual front end editor
4.  1st section full width section use full width slider module
5. Try to click on the tool tip of slider module

### Additional Information:

**Device**: Mac 

**Operating System**: Mac

**Browser**: Firefox

**Screenshot:

![screen shot 2016-09-09 at 6 58 53 am](https://cloud.githubusercontent.com/assets/4274448/18372318/7407c168-765b-11e6-8ba2-1f54f03ac2c0.png)

**Comment by fikrirasyid** [comment]
@rijendra can you share the layout used or access token to the site? We already have adjustment for this kind of scenario. It should look like this (it is working on my local site):

![intentional-space](https://cloud.githubusercontent.com/assets/916442/18381613/3bb92b16-76a7-11e6-9043-0ae655f9b8c6.jpg)

Here's how it looks like in different primary nav bg color:

![intentional-space-2](https://cloud.githubusercontent.com/assets/916442/18381652/71e0000c-76a7-11e6-8d14-068cdc4eaf86.jpg)


Basically, the adjustment works if:

1. primary nav uses rgba color
2. the first row is fullwidth module; the first fullwidth module's padding / margin will be adjusted
3. If the first row is regular section & module, the first section's padding will be adjusted.

**Comment by rijendra** [comment]
@fikrirasyid I tested this in 3.0.1 after this theme update and it seems to be fine now

All good now
Thanks
![screen shot 2016-09-12 at 9 56 04 pm](https://cloud.githubusercontent.com/assets/4274448/18443248/c18c78f4-7933-11e6-8f00-2be5e1a42ca5.png)
